src/main/java/org/mule/runtime/api/interception/InterceptionException.java

```
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.api.interception;
+
+/**
+ * Default exception thrown when a fail without cause is being called in an interceptionAction.
+ *
+ * @since 4.0
+ */
+public class InterceptionException extends RuntimeException {
```

Discussed this with Dan if we are to throw this with out action from the user, the user may want to work with the class, being API stop any CL exception. 
Also Dan pointed out we could throw any other exception but this is up to you guys to tell me what you want.